packages/dai/test/eth/MulticallService.spec.js

```diff
@@ -1,18 +1,28 @@
 import TestAccountProvider from '@makerdao/test-helpers/src/TestAccountProvider';
 import { buildTestMulticallService } from '../helpers/serviceBuilders';
+import schemas from '../helpers/schemas';
+import addresses from '../../../dai-plugin-mcd/contracts/addresses/testnet.json';
```

Comment:
this file should be about testing the general functionality of the multicall service. so the tests in this file should use an arbitrary schema, composed only for use in this test file, and any references to MCD should be in tests that live in the MCD plugin.

the fact that you're importing something directly from inside a plugin should be a good signal that something needs to change :)

Reply:
You're right. These tests will be changed to only rely on SCD state in the testchain, which will also remove the need to import and set the MCD addresses.

Alternatively I wonder if it's worth having some contracts and state in the testchain that's SCD and MCD agnostic and can be used to test multicall.